Makefile

```
@@ -0,0 +1,13 @@
+publish:
+	cd books && pdm run strawberry export-schema app > ../books.graphql
+	cd reviews && pdm run strawberry export-schema app > ../reviews.graphql
+
+	APOLLO_KEY=. \
+		rover subgraph publish strawberry-federation@current \
+		--name reviews --schema ./reviews.graphql \
+		--routing-url http://reviews:4000/graphql
+
+	APOLLO_KEY=. \
```

```suggestion
include .env

publish:
	cd books && pdm run strawberry export-schema app > ../books.graphql
	cd reviews && pdm run strawberry export-schema app > ../reviews.graphql

	APOLLO_KEY=$(APOLLO_KEY) \
		rover subgraph publish strawberry-federation@current \
		--name reviews --schema ./reviews.graphql \
		--routing-url http://reviews:4000/graphql

	APOLLO_KEY=$(APOLLO_KEY) \
```

Just learned: just putting `include` will do the trick!
Also the geraph name should be a variable maybe?